rest/config.go

```diff
@@ -1461,9 +1461,9 @@ func (sc *ServerContext) migrateV30Configs(ctx context.Context) error {
 		if insertErr != nil {
 			if insertErr == base.ErrAlreadyExists {
 				base.DebugfCtx(ctx, base.KeyConfig, "Found legacy config for database %s, but already exists in registry.", base.MD(dbConfig.Name))
-				continue
+			} else {
+				return fmt.Errorf("Error migrating v3.0 config for bucket %s groupID %s: %w", base.MD(bucketName), base.MD(groupID), insertErr)
```

Comment:
```suggestion
				return base.ReadactErrorf("Error migrating v3.0 config for bucket %s groupID %s: %w", base.MD(bucketName), base.MD(groupID), insertErr)
```

Reply:
Frustratingly `RedactableError` doesn't work with `%w`, as it uses `fmt.Sprintf` under the hood (which also doesn't support `%w`)

https://issues.couchbase.com/browse/CBG-2997

But I have just had an idea of how to make this work - will experiment and open PR if it works

Reply:
Good catch, there's an earlier error that should follow the same pattern, will fix both.